Modification de la documentation (installation backend et installatio…

Jars-of-jam-Scheduler opened the pull request:
…n frontend).

---> Quelques modifications sur les commandes Ubuntu
---> Ajout des commandes Fedora
---> Ajout de la commande gulp build (jusque-là, il n'était pas clairement demandé à l'utilisateur de la saisir, alors qu'elle est nécessaire s'il veut pouvoir consulter le site en local)
---> Commandes commentées (l'utilisateur doit-il être sudo ou pas, l'env. Python doit-il être activé ou pas, etc.)

Jars-of-jam-Scheduler committed:
Modification de la documentation (installation backend et installation frontend).
---> Quelques modifications sur les commandes Ubuntu
---> Ajout des commandes Fedora
---> Ajout de la commande gulp build (jusque-là, il n'était pas clairement demandé à l'utilisateur de la saisir, alors qu'elle est nécessaire s'il veut pouvoir consulter le site en local)
---> Commandes commentées (l'utilisateur doit-il être sudo ou pas, l'env. Python doit-il être activé ou pas, etc.)

Jars-of-jam-Scheduler committed:
Merge branch 'dev' of git://github.com/zestedesavoir/zds-site into dev

pquentin commented:
Merci, c'est toujours utile d'améliorer la doc et les instructions d'installations.

"si qq a màj" --> si quelqu'un a mis à jour

Pourquoi cette obsession avec sudo et l'environnement actif ? Je pense qu'on peut plus simplement mettre `$ sudo` devant les commandes qui nécessitent sudo, et `(zdsenv) $` devant le commandes qui doivent être dans le virtualenv. Non ?

Jars-of-jam-Scheduler committed:
Corrections :
"si qq a màj" devient "si quelqu'un a mis à jour"
Dans le fichier "frontend" : suppression du commentaire "Pour installer nodejs sous Fedora", remplacé par une commande.

Jars-of-jam-Scheduler committed:
Revert "Merge branch 'dev' of git://github.com/zestedesavoir/zds-site into dev"

This reverts commit dd6b9f1b767fb1fbc793ed1f9019e4a6c3641029, reversing
changes made to 30ce8c6b73143cf42a4e7a501f47670a4dbe046f.

Jars-of-jam-Scheduler commented:
@pquentin : j'ai eu des soucis à l'installation de l'environnement de travail parce que ces indications n'étaient pas précisées. Les commentaires permettent d'éviter à coup sûr ces problèmes. Un "Ne pas être sudo" c'est clair, et ça garantit que la commande passera !

pierre-24 commented:
Oups, y'a un _rebase_ qui a du foirer quelque part, cette PR modifie des choses qu'elle n'est pas censée modifier :o

Jars-of-jam-Scheduler commented:
Salut @pierre-24 , j'ai visiblement fait une fausse manip', je m'en occupe !

gustavi commented:
On en est où @Lern-X ?
